meetings/2021-06-01-meeting-minutes.md

```diff
@@ -0,0 +1,36 @@
+# Meeting agenda: Tuesday June 1, 2021
+
+## Review action items
+
+- [ ] Tom to share information on other groups mentor programs. *In progress*
+- [X] Bill Wagner to send emails to all selected Nominating Committee members.
+- [ ] Bill Wagner and Beth Massi will partner to create a Board expectations document for upcoming elections. *In progress*
+- [X] Bill Wagner to connect with Iris Classon and Rebeb Othmani to start connecting with the other Nominating Committee members.
+- [ ] Bill Wagner to send the Membership Committee’s budget wish list to the Finance team. *In progress*
+
+## Budget discussion
+
+The Membership Committee discussed their budget wish list. The Committee agreed to ask for a budget of $15,000. The following wish list items were discussed:
+
+- Define tooling to build a community.
+- Create career development opportunities.
+- Better member benefits awareness
+- Promoting diversity within the organization
+- Sponsor members to attend conferences and code camps. 
+- Offer a childcare credit to members who attend certain events.
+
+### [#21 - Determine the value proposition for membership](https://github.com/dotnet-foundation/wg-membership/issues/21)
+
+The Membership Committee held a discussion on the .NET Foundation’s value proposition. The following ideas were discussed:
+
+- Create “How to” guides for promoting the Foundation’s projects and encourage participation. 
+- Promote the new member participation types and their benefits on the website.
+- Promote current member contributions from the community.
+- Encourage more corporate member contributions to the Foundation to offset Microsoft’s contributions. 
+- Promote ways members can get involved in the community that can be rewarded.
+- Teach members how to find a “good” issue or PR. 
```

Comment:
Look into creating a "standard" way for projects to highlight "external" contributors in the last month. dot.net/thanks but  for those thanks you's now listed under powertoys/microsoft terminal releases